pkg/apis/jaegertracing/v1/jaeger_types.go

```diff
@@ -431,16 +431,33 @@ type JaegerCassandraCreateSchemaSpec struct {
 	// +optional
 	Enabled *bool `json:"enabled,omitempty"`
 
+	// Image specifies the docker container to use to create the cassandra schema.
```

Comment:
Sorry, not `docker container` but `container image` :)

Reply:
This is not a blocker, though. Would you prefer me to merge this as is, or would you want to get this change in ?

Reply:
Oh, what a misunderstanding :D I'll fix that for sure!